draft-ietf-tls-tls13.md

```diff
@@ -4375,6 +4375,109 @@ currently deployed and properly configured TLS implementations.
 %%### Ticket Establishment
 %%### Updating Keys
 
+# State Machine
+
+This section provides a summary of the legal state transitions for the
+client and server handshakes. State names have no formal meaning but
+are provided in all capitals (e.g., START) for ease of comprehension.
```

Comment:
The first several times I read this sentence, I thought you were saying they were in all capitals for ease of comprehension. This was very entertaining but probably not what you meant. :-) Perhaps:

State names (in all capitals, e.g., START) have no formal meaning but are provided for ease of comprehension.